### PR TITLE
20200325 release

### PR DESCRIPTION
**Pull request by lscheske**
## Review checklist template

- [x] Correct Markdown and formatting
- [ ] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

-----
[View rendered activities/codebase-stewardship/activities.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/activities.md)
[View rendered activities/codebase-stewardship/create-assessment-spreadsheet.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/create-assessment-spreadsheet.md)
[View rendered activities/codebase-stewardship/criteria-for-codebase-stewardship.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/criteria-for-codebase-stewardship.md)
[View rendered activities/codebase-stewardship/for-existing-projects.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/for-existing-projects.md)
[View rendered activities/codebase-stewardship/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/index.md)
[View rendered activities/codebase-stewardship/lifecycle.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/lifecycle.md)
[View rendered activities/codebase-stewardship/odoo-codebase-template.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/odoo-codebase-template.md)
[View rendered activities/codebase-stewardship/open-assessment.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/open-assessment.md)
[View rendered activities/codebase-stewardship/stewardship-proposal-template.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/stewardship-proposal-template.md)
[View rendered activities/codebase-stewardship/supporting-governance/governance-template.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/supporting-governance/governance-template.md)
[View rendered activities/codebase-stewardship/supporting-governance/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/codebase-stewardship/supporting-governance/index.md)
[View rendered activities/events/attending-events.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/events/attending-events.md)
[View rendered activities/events/deciding-to-attend-events.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/events/deciding-to-attend-events.md)
[View rendered activities/events/process-contacts.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/events/process-contacts.md)
[View rendered activities/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/index.md)
[View rendered activities/member-relations/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/member-relations/index.md)
[View rendered activities/membership-growth/expression-of-interest.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/expression-of-interest.md)
[View rendered activities/membership-growth/founding-membership-proposal.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/founding-membership-proposal.md)
[View rendered activities/membership-growth/founding-membership-workshop.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/founding-membership-workshop.md)
[View rendered activities/membership-growth/identify-potential-codebase.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/identify-potential-codebase.md)
[View rendered activities/membership-growth/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/index.md)
[View rendered activities/membership-growth/information-about-founding-membership.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/information-about-founding-membership.md)
[View rendered activities/membership-growth/member-onboarding-support.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/member-onboarding-support.md)
[View rendered activities/membership-growth/membership-contract.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/membership-growth/membership-contract.md)
[View rendered activities/staff-management/travel.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/staff-management/travel.md)
[View rendered activities/staff-meetings/event-debrief.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/staff-meetings/event-debrief.md)
[View rendered activities/staff-meetings/index.md](https://github.com/publiccodenet/about/blob/20200325-release/activities/staff-meetings/index.md)

### CODE REVIEW

File: activities/codebase-stewardship/supporting-governance/governance-template.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+type: Resource
+---
+
+# Template for a simple governance file
+
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/supporting-governance/governance-template.md]: This template is missing context, which means it's not self-explanatory or usable if someone finds it on its own. Here's an example of a template where we have provided context: 
https://github.com/publiccodenet/about/blob/develop/activities/codebase-stewardship/assessment-issue-template.md

Comment (ElenaFdR) [reply activities/codebase-stewardship/supporting-governance/governance-template.md]: @Ainali - can you add context?

File: activities/events/attending-events.md [hunk 2]
```diff
@@ -17,23 +17,28 @@ Before using this guide, make sure to [get a decision on attending the event](de
...
 * Create internal awareness. In order to enable synergies, sharing the overall plans with the rest of the organization is important.
-* Check the agenda. If someone happens to be connected to any of the speakers, consider reaching out.
-* Consider bringing informational material to give away to people that want to get to know us better.
+* Check the agenda. If someone happens to be connected to any of the speakers, consider reaching out. Coordination about this should start a month before the event or as soon as the program is released.
+* If the event has a call for papers or accepts submissions or ideas for talks, consider answering that, since it puts us in a prime position to present our [mission](../../organization/mission.md)
+* Consider bringing informational material to give away to people that want to get to know us better. Contact the organizer to find out if there are places to put these at.
 * Consider using a shared document to do the planning in, perhaps using [this template](events-planning-template.md).
+* Plan your [travels](../staff-management/travel.md).
+* Prepare a team chat (that is capable of notifications) if you are more than staff member attending.
```
Comment (lscheske) [inline activities/events/attending-events.md]:
```suggestion
* Prepare a team chat (that is capable of notifications) if you are more than one staff member attending.
```

File: activities/codebase-stewardship/supporting-governance/governance-template.md (hunk 1 above)
Comment (bvhme) [reply activities/codebase-stewardship/supporting-governance/governance-template.md]: Perhaps we should mark this for improvement and then continue, as this is better to solve in the develop than in the release branch probably.

File: activities/events/process-contacts.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Process contacts after an event
+
+## Add contacts to Odoo
+
+If you agreed to keep in touch with someone at the event, add them as a contact in Odoo to enable that connection. Do not enter more details than what was shared with you for that purpose.
+
+## Follow up
+
+The way to follow up obviously depends on what interaction you had with the person at the event. If you had a good conversation with someone or heard them giving a good talk, consider sending them a brief thank you. Everyone that gives us inspiration in some way furthers our mission and deserves to be recognized. Other follow ups could include more information about the Foundation or the Standard or whatever you talked about.
```
Comment (lscheske) [inline activities/events/process-contacts.md]:
```suggestion
The way to follow up depends on what kind of interaction you had with the person at the event. If you had a good conversation with someone or heard them giving a good talk, consider sending them a brief thank you. Everyone that gives us inspiration in some way furthers our mission and deserves to be recognized. Other follow-ups could include sharing more information on the Foundation for Public Code, the Standard for Public Code, or any other topic relevant to the conversation you had.
```

File: activities/membership-growth/identify-potential-codebase.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ---
 type: Resource
+redirect_from:
+    - activities/member-relations/identify-potential-codebase.md
 ---
 
 # Identifying potential codebases for Foundation stewardship
```
Comment (lscheske) [inline activities/membership-growth/identify-potential-codebase.md]:
```suggestion
# Identifying potential codebases for stewardship by the Foundation for Public Code
```

File: activities/staff-meetings/index.md [hunk 5]
```diff
@@ -1,9 +1,10 @@
 ---
 type: Index
-explains: This index contains the resources and guides for staff to run and participate in staff meetings.
 ---
 
-# Staff meetings at the Foundation for Public Code
+# Staff meeting templates
+
+All meetings have an agenda in the meeting calendar invitation. For common meetings that have to be planned often you can use the folling templates:
```
Comment (lscheske) [inline activities/staff-meetings/index.md]:
```suggestion
All meetings have an agenda in the meeting calendar invitation. For common meetings that have to be planned often you can use the following templates:
```

File: activities/codebase-stewardship/supporting-governance/governance-template.md (hunk 1 above)
Comment (Ainali) [reply activities/codebase-stewardship/supporting-governance/governance-template.md]: Resolved in #599

**Comment by ElenaFdR** [comment]
Not all new files are linked from the relevant folder's index. Some are only linked from the most relevant guidance.  I've created #601 to solve this elsewhere.

**Pull request merged**